qa/SidechainTestFramework/scutil.py

```diff
@@ -1219,6 +1219,133 @@ def create_certificate_for_alien_sc(mcTest, scid, mc_node, fePatternArray):
     assert_equal(True, cert in mc_node.getrawmempool())
     return cert
 
+# 10 ^ 10
```

Comment:
All the account-related methods and constant were moved to account/utils.py, so this part should be moved there again